pyt/baseline.py

```diff
@@ -0,0 +1,11 @@
+import json
+
```

Comment:
I didn't notice this before but from looking at the [Code Climate issues](https://codeclimate.com/github/python-security/pyt/pull/106) at the bottom of the PR, there are a couple of pep8 ones. Normally people have 2 lines between imports and the next class or function call like in [`expr_visitor.py`](https://github.com/python-security/pyt/blob/master/pyt/expr_visitor.py#L31-L34) for example.